gateway/src/main/java/io/camunda/zeebe/gateway/Gateway.java

```diff
@@ -152,6 +155,17 @@ private static NettyServerBuilder setNetworkConfig(final NetworkCfg cfg) {
         .permitKeepAliveWithoutCalls(false);
   }
 
+  private void submitLongPollingActor(final LongPollingActivateJobsHandler handler) {
```

Comment:
Hm... I don't agree here, the long polling handler is not only responsible for probing at a fixed rate. To my understanding, the main purpose of the long polling handler is to manage the "lifecycle" of any inflight activate jobs request, i.e., it "marks" a request as `active`, `toBeRepeated`, or `pending`, in addition, it ensures that a request times out according to the given `requestTimeout`.

Also, the method name `#submitLongPollingActor()` changed to `#submitActivateJobsActor()` with a commit later, because now the `RoundRobinActivateJobsHandler` delegates to an `actor` as well. So, when bootstrapping the `Gateway`, depending on the configuration, it will create either a `LongPollingActivateJobsHandler` or a `RoundRobinActivateJobsHandler`. Independent from the concrete activate jobs handler instance, an `actor` gets scheduled that is passed to the instance of the activate jobs handler.

I suggest changing the method name to `submitActorToActivateJobs(final Consumer<ActorControl>)` which should reflect better on what the implementation of the method does.